### PR TITLE
Add Undertale Yellow

### PR DESCRIPTION
**Pull request by JeodC**
# New Port for Undertale Yellow

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/utyellow/port.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "desc": "When a child named Clover sets out on a mission, they find themself deep within the Underground. In this foreign land, monsters lurk around every corner, but are they friend or foe?",
+    "inst": "Download the game from Gamejolt and add the contents to ports/utyellow/assets. Game is ready to play after a brief first-time setup.",
+    "genres": [
+      "rpg"
+    ],
+    "image": "screenshot.png",
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": []
```
Comment (kloptops) [inline ports/utyellow/port.json:23]: Missing comma:
```suggestion
    "reqs": [],
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Comment by JeodC** [comment]
I might mark this one experimental later.

File: ports/utyellow/Undertale Yellow.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    $controlfolder/xdelta3 -d -s "$GAMEDIR/game.droid" "$GAMEDIR/utyellow.xdelta" "$GAMEDIR/game2.droid"
+    rm -rf game.droid
+    rm -rf utyellow.xdelta
+    mv game2.droid game.droid
+fi
+    echo "Done! Loading game..." > $CUR_TTY
+fi
+
+# Exports
+export LD_LIBRARY_PATH="$GAMEDIR/lib:$GAMEDIR/libs":$LD_LIBRARY_PATH
```
Comment (Cebion) [inline ports/utyellow/Undertale Yellow.sh:80]: Isn't the ld_library export done above already?

File: ports/utyellow/port.json
Comment (Cebion) [inline ports/utyellow/port.json:1]: Is this a old json format template you're using? The one from the website has fields for inst_md and  desc_md.

Comment (JeodC) [reply ports/utyellow/port.json:1]: Yes I didn't know the website had a new json version.

**Review by kloptops — APPROVED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**